libs/wingsdk/src/target-tf-aws/platform.ts

```diff
@@ -8,6 +8,41 @@ export class Platform implements IPlatform {
   /** Platform model */
   public readonly target = "tf-aws";
 
+  public readonly parameters = {
+    $id: "tf-aws-parameters",
+    type: "object",
+    properties: {
+      "tf-aws": {
+        type: "object",
+        properties: {
+          vpc: {
+            type: "string",
+            enum: ["new", "existing"],
+          },
+          vpc_id: {
+            type: "string",
+          },
+          private_subnet_id: {
+            type: "string",
+          },
+          public_subnet_id: {
+            type: "string",
+          },
```

Comment:
Is there a way to add comments or descriptions to the properties?

Reply:
Yes good call, the standard way in Json schema is to use "$comment" Il add